lib/blocs/request_manager/online_request_manager.dart

```diff
@@ -155,14 +164,13 @@ class OnlineRequestManager implements RequestManager {
     if (response.statusCode == 200) {
       return await compute(_parseAd, utf8.decode(response.bodyBytes));
     } else if (response.statusCode == 404) {
-      print ("No ads found");
+      print("No ads found");
```

Comment:
Instead of using print, I would recommend using `developer.log`
https://flutter.dev/docs/testing/code-debugging#logging